high_performance_computing/hpc_openmp/05_hybrid_parallelism.md

```diff
@@ -0,0 +1,440 @@
+---
+name: Introduction to Hybrid Paralellism
+dependsOn: [high_performance_computing.hpc_openmp.04_synchronisation]
+tags: [hybrid APIs]
+---
+
+
+At this point in the lesson, we've introduced the basics you need to get out there and start writing parallel code using
+OpenMP. There is one thing still worth being brought to your attention, and that is *hybrid parallelism*.
+
+:::callout
+## The Message Passing Interface (MPI)
+
+In this episode, we will assume you have some knowledge about the Message Passing Interface (MPI) and that you have a 
+basic understand of how to paralleise code using MPI. If you're not sure, you can think of MPI as being like an OpenMP
```

Comment:
perhaps put the mpi material as a dependency of this section?

Reply:
I've added the MPI material as a dependency for this episode/section, but not sure it's working. At least, it's not showing as a dependency in the material graph that I can see? Perhaps there's a better way to specify it?

Reply:
hmm, maybe this kind of dependecy isn't shown in the graph. I'll merge this in for now and we can look at it later.